mobile/lib/entities/asset.entity.dart

```diff
@@ -524,7 +525,8 @@ bool isRotated270CW(int orientation) {
 
 /// Returns `true` if this [Asset] is flipped 90° or 270° clockwise
 bool isFlipped(AssetResponseDto response) {
-  final int orientation = response.exifInfo?.orientation?.toInt() ?? 0;
+  final int orientation =
+      int.tryParse(response.exifInfo?.orientation ?? '0') ?? 0;
```

Comment:
You don't need `?? '0'`

Reply:
I do, per Dart :P 
![image](https://github.com/user-attachments/assets/fc0f6be1-0ed0-4eb0-b1df-46567aabb013)

Reply:
> Do you need both ?? '0' and ?? 0?

Yes, because `tryParse` can return `null` when it cannot parse the value